package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "svg-as-symbol-loader",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Do not manually change version here. It will be bumped during publishing automatically.